contractsSol5/KyberMatchingEngine.sol

```diff
@@ -191,6 +192,10 @@ contract KyberMatchingEngine is KyberHintHandler, IKyberMatchingEngine, Withdraw
         tData.tokenToEth.decimals = srcDecimals;
         tData.ethToToken.decimals = destDecimals;
         tData.networkFeeBps = info[uint(IKyberMatchingEngine.InfoIndex.networkFeeBps)];
+        tData.platformFeeBps = info[uint(IKyberMatchingEngine.InfoIndex.platformFeeBps)];
```

Comment:
to avoid long function

please create function initAndValidateTradeData

so we can have TradeData memory tData = initAndValidateTradeData(.,,)
and pass required values.

Reply:
Do we need the checks? Or are we shifting the checks from network contract to this one? Will feel like unnecessary duplicate checks if we do it in both places.

Reply:
This is an external view function, it can be called by normal user, so I think it is ok to check it. 
And it will make sense that other math calculation is not overflow with assumption that `networkFeeBps` and `platformFeeBps` less than 10000